.github/workflows/test.yaml

```diff
@@ -25,19 +25,29 @@ on:
 
 jobs:
   test:
-    # NOTE: The worker environment running this job needs firefox and
-    #       geckowebdriver available. Before upgrading to ubuntu-24.04 in the
-    #       future, check and see its available in a location like this:
-    #       https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#browsers-and-drivers
-    #
-    runs-on: ubuntu-22.04
-
+    timeout-minutes: 30
     strategy:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.11"]
         jupyter_server-version: ["1", "2"]
         jupyterlab-version: ["3"]
+        os: [ubuntu-22.04]
+        include:
+          - python-version: "3.11"
+            jupyter_server-version: "2"
+            jupyterlab-version: "3"
+            os: windows-2022
+
+    # NOTE: The worker environment running this job needs firefox and
+    #       geckowebdriver available. Before upgrading to ubuntu-24.04 in the
+    #       future, check and see its available in a location like this:
+    #       https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#browsers-and-drivers
+    #
```

Comment:
```suggestion
```
I think so since the acceptance tests seem to run OK on Windows. On the other hand I think it's safe to assume Firefox and Geckodriver won't be removed in newer OS images, and if they are we'll find out soon enough